src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -162,78 +164,63 @@ These events are also available for querying, creating custom dashboards, and al
-
-**Correlation**: Anomalies are connected to our correlation logic, otherwise known as decisions. If our correlation logic determines that issues are connected, then a single issue will be created from multiple incidents. You can see this correlated issue in the issues feed. Anomalies provide additional context about what changed around the time an issue occurred so your team can easily detect patterns and diagnose problems. 
-
-**Receiving notifications:** When New Relic detects anomalous changes in throughput, error rate, or response time, your team has the option to receive notifications about this behavior. We recommend sending notifications selected Slack channels or email channels. When the anomaly goes back to normal then a recovery message is sent. If you don't want to receive notifications you can still access to the data using a [NRQL query](#anomaly-events).
+## Anomaly set-up [#set-up]
 
-**Anomaly analysis:** For each anomaly, we provide a link in Slack to an analyze anomaly page. This page generates automatic insights into the anomaly. The page is also available from the anomalies tab, which lists recent anomalies. This page uses your existing APM and anomaly detection data to provide explanations as to the cause of the anomaly.
+Once you choose to monitor anomalous behavior in your system using either our custom or automatic anomaly detection, you will need to make sure that your team is notified of any unusual behavior and that you can query and understand your data. It doesn't matter if you choose custom or automatic anomaly detection– the set-up is the same. 
 
-**Activity stream:** Inside various activity streams such as the New Relic homepage, APM Summary page, Lookout and Explorer, you'll see relevant anomalies from your APM-monitored applications. Clicking on any of the anomaly events in the activity stream brings up the analysis page for that anomaly.
-
-Applications will not always generate anomalies, so it can be normal to not receive any detections.
-
-### Requirements [#requirements]
-
-To use New Relic's anomaly detection, ensure you have:
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="jira"
+    title="Requirements"
+  >
 
-* An APM agent installed for at least one application.
-* To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
+  To use New Relic's anomaly detection, ensure you have:
 
-For more on data limits, see [Data limits](#data-limits).
+  * An APM agent installed for at least one application.
+  * To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
```

Comment:
I think this line may need a slight tweak to work with the list format. If the reader reads each point as part of the "ensure you have" lead-in, this one doesn't make sense with that. Very small thing though, may not be worth changing.